ruby/vampires.rb

```diff
@@ -0,0 +1,152 @@
+
+def vampires
+
+puts "Welcome to Immortal Life Technologies"
+puts""
+
+counter = 0
+puts "How many of the following questionnaires will be administered?"
+total_employees = gets.chomp.to_i
+	until counter == total_employees
+	puts "Please fill out the following questionnaire to better serve your needs."
+	puts ""
+		
+	puts "What was you full name given at the time of your birth?"
+	birth_name = gets.chomp.capitalize
+	puts ""
+		
+	puts "How many years have you roamed this planet?"
+	age = gets.chomp.to_i
+	puts ""
+
+	puts "What was the year of your birth?"
+	birth_year = gets.chomp.to_i
+	puts ""
+
+# Mortal Age Confirmation #
+
+	mortal_age = false
+	until mortal_age
+
+#	puts "How many years have you roamed this planet?"
+#	age = gets.chomp.to_i
+
+#	puts "What was the year of your birth?"
+#	birth_year = gets.chomp.to_i
+
+		current_year = 2017
+		if age == (current_year - birth_year)
+			mortal_age = true
+		else
+			mortal_age = false
+		end
+	end
+
+# Garlic Aversion Confirmation #
+
+	garlic_choice = nil
+	while garlic_choice == nil
+
+	puts "Today the cafeteria is serving spaghetti. Would you care for some garlic bread? (y/n)"
+	garlic = gets.chomp.downcase
+	puts ""
+
+		if garlic == "yes" || garlic == "y"
+			garlic_choice = true
+		elsif garlic == "no" || garlic == "n"
+			garlic_choice = false
+		else
+			garlic_choice = nil
+		end
+	end
+
+# Immortality Confirmation #
+
+	mortality = nil
+	while mortality == nil
+
+	puts "Would you care to enroll in the Company's Health Insurance plan?"
+	health = gets.chomp
+	puts ""	
+
+		if health == "yes" || health == "y"
+			mortality = true
+		elsif health == "no" || health == "n"
+			mortality = false
+		else
+			mortality = nil
+		end
+	end
+
+# Vampire Booleans #
+
+#	V1 = vampire 
+
+    vampire = nil
+	while vampire == nil
+
+		if name == "Drake Cula"
```

Comment:
'Match the following conditions in the order they're listed.' We first read this when starting off the challenge and we may try to hack it out with a basic if/elsif structure but we quickly learn that the first condition to pass will be the result even if a 'more true' version is below.  We may be tempted to switch the order for a working solution, but if we are to follow the constraints we need to be more creative. What if we made each individual scenario it's own respective if block? What would go inside each if block? Surely we can't just puts out the result otherwise our screen would be full of each vampire check that passed. What we could do is keep setting a variable result within each if block and then at the end of all the blocks display the result to look something like this
```
if condition_1
	result = 'low'
end

if condition_2
	result = 'medium'
end

if condition_3
	result = 'high'
end

puts result
```